modules/form/main/index.scss

```diff
@@ -11,17 +11,26 @@
       display: table-cell;
     }
 
-    > *:not(input) {
+    > *:not(input):not(select):not(textarea):not(button):not(.hx-btn) {
```

Comment:
You know, I've never actually thought to try doing multiple selectors inside a `:not` - I've no idea if it works.